OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -164,6 +167,8 @@ - (instancetype)initWithName:(NSString *)name {
     
     if (self = [super init]) {
         _name = name;
+        _internalConnectionCount = 0;
+        _connectionsOlderThanMe = 0;
```

Comment:
default value will be `0` for `NSUInteger`, so this assignment is not necessary.